example.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Leaflet Geonames Search Control</title>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.0-rc.1/leaflet.css"/>
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="L.Control.Geonames.css"/>
+</head>
+<body>
+<script src="//cdnjs.cloudflare.com/ajax/libs/leaflet/1.0.0-rc.1/leaflet.js"></script>
+<script src="L.Control.Geonames.js"></script>
+<div id="map" style="position: absolute; top: 0; left:0; bottom:0; right: 0;"></div>
+
+<script>
+    var map = L.map('map').setView([44.07, -118.78], 6);
+    L.tileLayer('//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+        attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+        maxZoom: 10,
+        minZoom: 2
+    }).addTo(map);
+
+    var stateField = L.Control.extend({
+        options: {
+            position: 'topleft'
+        },
+        onAdd: function () {
+            this._container = L.DomUtil.create('div', 'leaflet-state-select leaflet-bar');
+            this._container.title = 'Select a state';
+            this._input = L.DomUtil.create('input', '', this._container);
+            this._input.id = 'state';
+            this._input.type = 'text';
+            this._input.placeholder = 'Enter a state';
+            return this._container;
+        }
+    });
+    map.addControl(new stateField());
+
+    var control = L.control.geonames({
+        username: 'your-geonames-username',
+        countries: 'us,ca',
```

Comment:
Can you add this as an text entry too?  This way results will be more obvious and makes it easier to merge with `index.html`.  (meaning: if user does not enter country or adminCode1, then results are equivalent to current version).